### ISSUE TITLE
Mobile: Tabbed Explore Section 

### ISSUE THREAD

**Issue by zakhap** [hicommonwealth/commonwealth#6537]
## Description

**Tabbed Explore Section**

- Update Tabs with new #5303
  - Behavior is sticky!
<img width="325" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/5334557/4c7162bc-34eb-4841-b612-08796a90b36e">

- On Scroll Behavior should include stickied tabs
<img width="266" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/5334557/62b3fc6b-eb35-4cd3-9ba4-d2e7f65958b0">

**Behavior of Interacting with Feed Cards**
- Feed cards should be use new designs! 
- As a user, when I click the comment icon, I should go to the thread page with the composer open (implemented on Discussions listing page

Devlink: https://www.figma.com/file/w6Tli9ApqBqgsZa7kB0Vk5/Mobile-ideation-Q4?type=design&node-id=761-52092&mode=dev

## Project Owner

PM: @sachben91

**Comment by rbennettcw** [comment]
Estimating as 2.

The remaining scope of new work seems to include:
- Sticky tabs
- Feed card redesign

To confirm, the horizontal communities carousel will *not* be added?

As for upvoting– it looks like there's nothing new to do there?

cc @zakhap @sachben91

**Comment by zakhap** [comment]
> To confirm, the horizontal communities carousel will not be added?

Correct, that is a separate ticket!

**Comment by rbennettcw** [comment]
> > As for upvoting– it looks like there's nothing new to do there?
> 
> @rbennettcw We don't currently have support for reacting/upvoting on the feedback as we don't have an active address at that scope. Does that change your estimation?

@zakhap  Can you point me to the design/screenshot that illustrates this? In the ticket, those screenshots don't show any upvote buttons. Trying to understand what all is in scope.

**Comment by zakhap** [comment]
ah, I see. 

Let's dismiss that part of the ticket for the moment. Will update above.

**Comment by mzparacha** [comment]
> Feed cards should be use new designs!

do we have the new designs?

cc: @zakhap

**Comment by mzparacha** [comment]
Added scroll in the draft PR https://github.com/hicommonwealth/commonwealth/pull/6888, blocked on [this](https://github.com/hicommonwealth/commonwealth/issues/6537#issuecomment-1964254050)

**Comment by zakhap** [comment]
The designs are here: https://www.figma.com/file/w6Tli9ApqBqgsZa7kB0Vk5/Mobile-ideation-Q4?type=design&node-id=761-52092&mode=dev
